plugins/modules/dcnm_service_route_peering.py

```diff
@@ -0,0 +1,3707 @@
+                        )
+                        self.module.fail_json(msg=mesg)
+
+                    in_list.remove(rt)
+                    rt.update(in_net_route[0])
+
+            if item.get(net_name2, "") != "":
+                if item[net_name2]["profile"].get("static_route", "") != "":
+                    # Static Route is a list of route dicts
+                    for rt in item[net_name2]["profile"]["static_route"]:
+                        out_list.append(rt)
+                        out_net_route, invalid_params = validate_list_of_dicts(
+                            out_list, srp_static_route_spec
+                        )
+                        if invalid_params:
+                            mesg = "Invalid parameters in playbook: {}".format(
+                                "while processing Static Route under Network/Arm - "
+                                + net_name2
+                                + ", under Route Peering - "
+                                + cfg[0]["name"]
+                                + ", "
+                                + "\n".join(invalid_params)
+                            )
+                            self.module.fail_json(msg=mesg)
+
+                        out_list.remove(rt)
+                        rt.update(out_net_route[0])
+
+    #############################################################################################
+    # Routine to validate the given playbook input based on the type of peering
+    #
+    # Input:
+    #        self	- SRP class object containing the playbook config and all payload information
+    #
+    # Output:
+    #        self.srp_info	- SRP information populated with appropriate data after validation
+    #
+    # Returns:
+    #        None
+    #############################################################################################
+
+    def dcnm_srp_validate_input(self):
+
+        if None is self.config:
+            return
+
+        # Inputs will vary for Firewall and ADC service nodes and for each state. Make specific checks
+        # for each case.
+
+        cfg = []
+        for item in self.config:
+
+            citem = copy.deepcopy(item)
+
+            cfg.append(citem)
+
+            if self.module.params["state"] == "deleted":
+                # config for delete state is different. So validate deleted state differently
+                self.dcnm_srp_validate_delete_state_input(cfg)
+            elif self.module.params["state"] == "query":
+                # config for query state is different. So validate query state differently
+                self.dcnm_srp_validate_query_state_input(cfg)
+            # For 'overridden' state, we can have full config for a peering or just service node name alone.
+            # In the formar case go down to 'else' block to validate the full config
+            elif (self.module.params["state"] == "overridden") and (
+                item.get("name", None) is None
```

Comment:
I believe `item.get` will automatically return `None` if there is not value so not sure you need to default to `None`

Reply:
Some places I use a different value than None. So to make it consistent I use this style